<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MADMARDIGAN" name="team.boolbee.hibernate.xml.manytomany.unidir.model.ManyToManyBidirXmlTest" tests="2" failures="1" timestamp="20 Nov 2018 10:08:48 GMT" time="1.332" errors="1">
  <testcase name="testDeleteCustomer" time="0.000" classname="team.boolbee.hibernate.xml.manytomany.unidir.model.ManyToManyBidirXmlTest"/>
  <testcase name="testBidirectionalManyToManyAsociation" time="1.332" classname="team.boolbee.hibernate.xml.manytomany.unidir.model.ManyToManyBidirXmlTest">
    <error type="java.lang.AssertionError" message="expected [null] but found [Customer [id=5, name=Mark Zuckerberg, address=Palo Alto, California, Estados Unidos, bankAccounts=[BankAccount [id=9, number=4100-4400-4404-4404-0002, deposit=3000000.0], BankAccount [id=8, number=4100-4400-4404-4404-0001, deposit=3000000.0]]]]">
      <![CDATA[java.lang.AssertionError: expected [null] but found [Customer [id=5, name=Mark Zuckerberg, address=Palo Alto, California, Estados Unidos, bankAccounts=[BankAccount [id=9, number=4100-4400-4404-4404-0002, deposit=3000000.0], BankAccount [id=8, number=4100-4400-4404-4404-0001, deposit=3000000.0]]]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotSame(Assert.java:490)
	at org.testng.Assert.assertNull(Assert.java:426)
	at org.testng.Assert.assertNull(Assert.java:415)
	at team.boolbee.hibernate.xml.manytomany.unidir.model.ManyToManyBidirXmlTest.testBidirectionalManyToManyAsociation(ManyToManyBidirXmlTest.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testBidirectionalManyToManyAsociation -->
</testsuite> <!-- team.boolbee.hibernate.xml.manytomany.unidir.model.ManyToManyBidirXmlTest -->
